apps/src/studio/blocks.js

```diff
@@ -3405,250 +3424,6 @@ exports.install = function (blockly, blockInstallOptions) {
     );
   };
 
-  //
-  // Install functional start blocks
```

Comment:
The deleted blocks below this point were only used in CS in Algebra, which has been deprecated.